src/BrutForceForCaesarCipher.java

```diff
@@ -0,0 +1,148 @@
+import AlphabetsForCipher.Alphabet;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Scanner;
+
+public class BrutForceForCaesarCipher {
+    private static Alphabet alphabet;
+    private static final String MANUAL_BRUT_FORCE = "manual";
+    private static final String BRUT_FORCE_WITH_ANALYZER = "analyzer";
+
+    public static void startBrutForce(Alphabet setAlphabet){
+        alphabet = setAlphabet;
+        int usingAlphabetSize = setAlphabet.getAlphabetCodes().size();
+        System.out.println("BrutForce starting...");
+        //Get file for decode
+        System.out.println("Enter path to file which you want to decode with BrutForce");
+        Path path = WorkWithFiles.getFilePath();
+
+        //choose brut method
+        String brutForceMethod = choiseOfBrutForceMethod();
+
+        //Read data from file
+        StringBuilder decodedDataFromFile = new StringBuilder();
+        if (brutForceMethod.equals(MANUAL_BRUT_FORCE)){
+             decodedDataFromFile = startManualBrutForce(path, usingAlphabetSize);
+        } else if (brutForceMethod.equals(BRUT_FORCE_WITH_ANALYZER)){
+            decodedDataFromFile = startBrutForceWithAnalyzer(path, usingAlphabetSize);
+        } else {
+            //TODO make this exception
+            //throw invalidBrutMethodException;
+        }
+        
+        //Create and get path to file
+        Path pathToEncodingFile = WorkWithFiles.createFile(path, "Decoded_by_BrutForce");
+
+        //Write data to file
+        try {
+            Files.writeString(pathToEncodingFile, decodedDataFromFile);
+            System.out.printf("Decoded file created: %s %n", pathToEncodingFile.getFileName().toString());
+        } catch (IOException e) {
+            System.out.println("Check that the coding file in encoding UTF-8");
+            throw new RuntimeException(e);
```

Comment:
it's better to create your custom exception

Reply:
Why? There ara a lot of standart java exceptions for this case
I think maybe i need to use "CharacterCodingException" with this custom message in first case and IOException in second